### DISCUSSION TITLE
What's the difference between light and heavy portable?

### DISCUSSION THREAD

**Discussion by ElektroStudios** [NickeManarin/ScreenToGif#1168] [Q&A] [answered]
As the title says: What's the difference between light and heavy portable?, I don't see it mentioned in the README.MD.

Also, the optional 'gifsky.dll' it is needed for the heavy-size portable or it comes embedded with it?.

The last version I used it was 2.24.2, which is a small file of around 3,65 MB, so this new light/heavy portable choice is totally new for me...

Thanks in advance.

**Comment by NickeManarin** [comment, marked as answer]
Hi, the "light" version doesn't have .NET 6 along its binaries. 
Meaning that you have to install it separately.

Previous versions were all "light", but since most of them used .NET Framework instead of .NET, it came installed with Windows already.
